Remove atk-bridge support. Modified for KaNaPi.

atk-bridge doesn't seem useful for now in Buildroot and requires to
add two new packages just for it: at-spi2-core and at-spi2-atk.

Signed-off-by: Hadrien Boutteville <hadrien.boutteville@gmail.com>
Modified-by: Jacek Danecki <jacek.m.danecki@gmail.com>

--- a/gtk/a11y/gtkaccessibility.c
+++ b/gtk/a11y/gtkaccessibility.c
@@ -37,10 +37,6 @@
 #include <gtk/gtkcombobox.h>
 #include <gtk/gtkaccessible.h>

-#ifdef GDK_WINDOWING_X11
-#include <atk-bridge.h>
-#endif
-
 static gboolean gail_focus_watcher      (GSignalInvocationHint *ihint,
                                          guint                  n_param_values,
                                          const GValue          *param_values,
@@ -987,9 +983,5 @@
   _gtk_accessibility_override_atk_util ();
   do_window_event_initialization ();

-#ifdef GDK_WINDOWING_X11
-  atk_bridge_adaptor_init (NULL, NULL);
-#endif
-
   atk_misc_instance = g_object_new (GTK_TYPE_MISC_IMPL, NULL);
 }
